### DISCUSSION TITLE
Doesn't respect uppercase elements

### DISCUSSION THREAD

**Discussion by patrickdorival** [downforcetech/fluidxml.php#21] [Questions]
Doing something like this (agentID converts to agentid)
```
$xml = fluidxml('propertyList');
$xml->add("agentID", $property->agent_id);
```
results in this
```
<propertylist>
    <agentid>10173</agentid>
</propertylist>
```

Also how do I add attributes directly to the root element?  
I can do it with 

```
$xml->query('//propertyList')
	->attr('date', "{$today}");
```

But I just thought that it should probably be something that you can do when you instantiate.

**Comment by patrickdorival** [comment]
My mistake, it was echoing it out that way, saving the xml revealed 

```
<?xml version="1.0" encoding="UTF-8"?>
<propertyList date="2016-11-21-11:48:13">
  <rental modTime="2016-11-09 13:05:41" status="leased">
    <agentID>10173</agentID>
  </rental>
</propertyList>

```

So I suppose the only question is the root element attributes.

**Comment by daniele-orlando** [comment]
```$xml->attr(...);```
